<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulatorSettingsDialog</class>
 <widget class="QDialog" name="SimulatorSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulator Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>&amp;General</string>
      </attribute>
      <widget class="QCheckBox" name="hideSimulatorMenuCheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>301</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Hide Simulator Settings &amp;Menu Item From Users</string>
       </property>
       <property name="shortcut">
        <string>Alt+M</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="administratorAccountRequiredCheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>195</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>&amp;Administer Account Required</string>
       </property>
       <property name="shortcut">
        <string>Alt+A</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="diskCacheGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>241</width>
         <height>161</height>
        </rect>
       </property>
       <property name="title">
        <string>Disk Cache</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QRadioButton" name="noCacheRadioButton">
          <property name="text">
           <string>No Cache</string>
          </property>
          <attribute name="buttonGroup">
           <string>diskCacheButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="forceCacheRadioButton">
          <property name="text">
           <string>Force Cache</string>
          </property>
          <attribute name="buttonGroup">
           <string>diskCacheButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="autoCacheRadioButton">
          <property name="text">
           <string>Auto Cache</string>
          </property>
          <attribute name="buttonGroup">
           <string>diskCacheButtonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tabNavigator">
      <attribute name="title">
       <string>Navigator</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0" colspan="4">
        <widget class="QCheckBox" name="emulateOpticalDiskCheckBox">
         <property name="text">
          <string>Emulate &amp;Optical Disk</string>
         </property>
         <property name="shortcut">
          <string>Alt+O</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QCheckBox" name="dualSidedDiskCheckBox">
         <property name="text">
          <string>Dual Sided Disk</string>
         </property>
         <property name="shortcut">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QSpinBox" name="emulatedOpticalDiskCapacitySpinBox">
         <property name="maximum">
          <number>1028</number>
         </property>
         <property name="singleStep">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLabel" name="textLabel1_2_2">
         <property name="text">
          <string>Emulated Optical Disk Capacity 
(MB, &amp;0 = unlimited)</string>
         </property>
         <property name="buddy">
          <cstring>emulatedOpticalDiskCapacitySpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" colspan="4">
        <widget class="QCheckBox" name="oldStyleNavigatorCheckBox">
         <property name="text">
          <string>Old St&amp;yle Navigator</string>
         </property>
         <property name="shortcut">
          <string>Alt+Y</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QCheckBox" name="newStyleBlueBarCheckBox">
         <property name="text">
          <string>New Style Blue Bar</string>
         </property>
         <property name="shortcut">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="textLabel1">
         <property name="text">
          <string>La&amp;boratory Name</string>
         </property>
         <property name="buddy">
          <cstring>labNameLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="2" colspan="2">
        <widget class="QLineEdit" name="labNameLineEdit"/>
       </item>
       <item row="6" column="0" colspan="4">
        <widget class="QCheckBox" name="useLabNameCheckBox">
         <property name="text">
          <string>Use Lab Name In Network Catalog</string>
         </property>
         <property name="shortcut">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="4">
        <widget class="QCheckBox" name="permanentDeleteCheckBox">
         <property name="text">
          <string>&amp;Allow Permanent Deletion of Study Data</string>
         </property>
         <property name="shortcut">
          <string>Alt+A</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QCheckBox" name="bluePanelTweakCheckBox">
         <property name="text">
          <string>Fix Blue Bar Buttons</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabRecorder">
      <attribute name="title">
       <string>Recorder</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="emulateTwoScreensCheckBox">
         <property name="text">
          <string>Emulate 2 Screens</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="emulateOneScreenCheckBox">
         <property name="text">
          <string>Emulate 1 Screen</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="useTwoRecorderWindowsCheckBox">
         <property name="text">
          <string>Use 2 Windows</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="emulateWindowsManagerCheckBox">
         <property name="text">
          <string>Emulate CardioLab Windows Manager</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="emulatePruckaTilingCheckBox">
         <property name="text">
          <string>Tiling Using CardioLab Layout</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="immovablePatientStatusBarCheckBox">
         <property name="text">
          <string>Immovable Patient Status Bar</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="patientStatusBarHasTitleCheckBox">
         <property name="text">
          <string>Patient Status Bar Has Title</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="recorderHasStatusBarCheckBox">
         <property name="text">
          <string>Recorder Window Has Status Bar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabHardware">
      <attribute name="title">
       <string>Hardware</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QComboBox" name="amplifierTypeComboBox">
         <item>
          <property name="text">
           <string>32 Channel</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>48 Channel</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>64 Channel</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>96 Channel</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>128 Channel</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Amplifier Type</string>
         </property>
         <property name="buddy">
          <cstring>amplifierTypeComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>372</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>administratorAccountRequiredCheckBox</tabstop>
  <tabstop>hideSimulatorMenuCheckBox</tabstop>
  <tabstop>emulateOpticalDiskCheckBox</tabstop>
  <tabstop>dualSidedDiskCheckBox</tabstop>
  <tabstop>emulatedOpticalDiskCapacitySpinBox</tabstop>
  <tabstop>oldStyleNavigatorCheckBox</tabstop>
  <tabstop>newStyleBlueBarCheckBox</tabstop>
  <tabstop>bluePanelTweakCheckBox</tabstop>
  <tabstop>labNameLineEdit</tabstop>
  <tabstop>useLabNameCheckBox</tabstop>
  <tabstop>permanentDeleteCheckBox</tabstop>
  <tabstop>emulateTwoScreensCheckBox</tabstop>
  <tabstop>emulateOneScreenCheckBox</tabstop>
  <tabstop>useTwoRecorderWindowsCheckBox</tabstop>
  <tabstop>emulateWindowsManagerCheckBox</tabstop>
  <tabstop>emulatePruckaTilingCheckBox</tabstop>
  <tabstop>immovablePatientStatusBarCheckBox</tabstop>
  <tabstop>patientStatusBarHasTitleCheckBox</tabstop>
  <tabstop>recorderHasStatusBarCheckBox</tabstop>
  <tabstop>amplifierTypeComboBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="diskCacheButtonGroup"/>
 </buttongroups>
</ui>
